apps/rush/src/start.ts

```diff
@@ -36,19 +38,77 @@ const configuration: MinimalRushConfiguration | undefined =
 
 const currentPackageVersion: string = PackageJsonLookup.loadOwnPackageJson(__dirname).version;
 
-let rushVersionToLoad: string | undefined = undefined;
+let rushVersionToLoadInfo: {
+  version: string;
+  path?: string;
+} | undefined = undefined;
+
+let overrideInfo: {
+  version: string;
+  path: string;
+} | undefined = undefined;
+
+if (configuration) {
+  const rushOverrideFilePath: string = path.join(
+    path.dirname(configuration.rushJsonFilename),
+    rushLib.RushConstants.commonFolderName,
+    rushLib.RushConstants.rushTempFolderName,
+    '.rush-override'
+  );
```

Comment:
```suggestion
  const rushOverrideFilePath: string = `${configuration.rushJsonFolder}/.rush-override`;
```

As per discussion.

Also stick `.rush-override` in a RushConstants.